grid-client/deployer/tf_plugin_client.go

```diff
@@ -105,6 +105,7 @@ func NewTFPluginClient(
 	proxyURL string,
 	rmbTimeout int,
 	showLogs bool,
+	allowRMBCache ...bool,
```

Comment:
should be just bool not ...bool, and should be rmbMemCache and if false we use the disk cache